tools/lint/CMakeLists.txt

```diff
@@ -1,7 +1,12 @@
 # yanglint
 
+if(WIN32)
+    set(YANGLINT_INTERACTIVE OFF)
+else()
+    set(YANGLINT_INTERACTIVE ON)
+endif()
```

Comment:
This is adding a fair amount of complexity for `yanglint` compilation, I do not like this. But I am not sure how to improve it so I am let this one go.

Reply:
Thanks. The alternative would be bringing in "something like `linenoise`", but for Windows, and I do not really prefer that :).